lib/src/marker_cluster_layer.dart

```diff
@@ -174,7 +174,6 @@ class _MarkerClusterLayerState extends State<MarkerClusterLayer>
         onTap: _onMarkerTap(marker),
         onHover: (bool value) => _onMarkerHover(marker, value),
         buildOnHover: widget.options.popupOptions?.buildPopupOnHover ?? false,
```

Comment:
The removal of this line is not needed and will lead to undesired behaviour.
```suggestion
        buildOnHover: widget.options.popupOptions?.buildPopupOnHover ?? false,
        hoverOnTap: () => widget.options.onMarkerTap!(marker),
```